mlflow/models/docker_utils.py

```diff
@@ -84,6 +88,11 @@ def generate_dockerfile(
     install_mlflow_steps = _pip_mlflow_install_step(output_dir, mlflow_home)
 
     if base_image.startswith("python:"):
+        if install_java:
+            _logger.warning(
+                "`install_java` option is not supported when using python base image, "
+                "switch to UBUNTU_BASE_IMAGE to enable java installation."
```

Comment:
Should this be resolved to `ubuntu:20.04`?! 🤔 In other words, shouldn't it be `%s`? @serena-ruan @rgangopadhya